data_pipeline_api/pipeline.py

```diff
@@ -1,12 +1,58 @@
 import datetime
+import json
 import logging
 import os
 
+import requests
 import yaml
 
 from data_pipeline_api import fdp_utils
 
 
```

Comment:
Could we add a wrapper function for searching data_products e.g.: 
search_data_product(handle, wildcard)
    return search(handle, wildcard, "name", "data_product")
As I imagine this will be the common use case.

Reply:
ok